Bottom: 07c111717c34ed00818872098a4062a2c9d574d2
Top:    81b8742f27c0ab40c595978be4d6cf995cfa2e3a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 15:56:41 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index cc186c7..1b9fbce 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1019,10 +1019,8 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   min_x = MIN (min_x, new_x);  max_x = MAX (max_x, new_x);
   min_y = MIN (min_y, new_y);  max_y = MAX (max_y, new_y);
 
-  region.X1 = MIN (Px (min_x), Px (max_x + 1));
-  region.X2 = MAX (Px (min_x), Px (max_x + 1));
-  region.Y1 = MIN (Py (min_y), Py (max_y + 1));
-  region.Y2 = MAX (Py (min_y), Py (max_y + 1));
+  region.X1 = min_x; region.X2 = max_x + 1;
+  region.Y1 = min_y; region.Y2 = max_y + 1;
 
   glColor3f (port->bg_color.red / 65535.,
              port->bg_color.green / 65535.,
